runner/operator/access/v1_0_0/legacy/__init__.py

```diff
@@ -38,7 +55,18 @@
 in source or exist with an empty value.
 """
 def get_missing_fields(source, fields):
-    return list(filter(lambda field: field not in source or not source[field], fields))
+
+    def validate(field):
+        if field not in source:
+            return True
+        elif not source[field]:
+            return True
+        elif type(source[field]) == list:
```

Comment:
@aef- this is to handle the failure we had where just one of the fields in the `add_rg_PU: ['IDT234', None, ...]` field was null